src/rules/attr-whitespace.js

```diff
@@ -0,0 +1,44 @@
+HTMLHint.addRule({
+  id: 'attr-whitespace',
+  description:
+    'All attributes should be separated by only one space and not have leading/trailing whitespace.',
+  init: function(parser, reporter, options) {
+    var self = this;
+    var exceptions = Array.isArray(options) ? options : [];
+    //console.log(exceptions);
+    if (exceptions.length == 0) {
+    }
```

Comment:
What does this if-branch do? Was it needed for something that is no longer available?

Reply:
@Shinigami92 Thank you for your feedback on my code! I believe I have resolved all of the issues in my latest commit.